src/mds/MDLog.cc

```diff
@@ -1309,13 +1309,7 @@ void MDLog::_replay_thread()
   int r = 0;
   while (1) {
     // wait for read?
-    while (!journaler->is_readable() &&
-	   journaler->get_read_pos() < journaler->get_write_pos() &&
-	   !journaler->get_error()) {
-      C_SaferCond readable_waiter;
-      journaler->wait_for_readable(&readable_waiter);
-      r = readable_waiter.wait();
-    }
+    r = journaler->check_isreadable();
```

Comment:
This return value is unused. Should it be `void`?

Reply:
The return value is not accurate when it call "finisher->queue(onreadable, 0);" directly. So I think "return void" is ok too. I will change it later.